core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -1376,8 +1376,14 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
     }
 
     fun moveCapitalToNextLargest() {
-        moveCapitalTo(cities
-            .filterNot { it.isCapital() }
+        val availableCities = cities.filterNot { it.isCapital() }
+        if (availableCities.filterNot { it.isPuppet }.isEmpty()) {
```

Comment:
What if there are no other cities at all? This function assumes that if there are no other non-puppet cities, there's at least one puppet city